test/rules/cascade_invocations.dart

```diff
@@ -40,9 +59,10 @@ class Foo {
 void noCascadeWithGetter() {
   final foo = new Foo();
   foo.baz(); // LINT
-  foo.bar; // LINT
-  foo.foo(); // LINT
+  foo.bar; // OK
+  foo.foo(); // OK
   foo.bar = 8; // LINT
```

Comment:
The original intention of the change was to lint things that looked like builder invocations, but since I couldn't find the code that I had difficulties with when I tried to enable the lint in my project we agreed on expanding the scope. That said, this now should be linted too.